languages/swift/exercises/concept/tuples/.docs/instructions.md

```diff
@@ -0,0 +1,37 @@
+You are an elf working in Santa Claus' logistics division and you have been given a pair of tasks from the boss for upgrading the system's software.
+
+## 1. Convert coordinates to polar
+
+Your first task is updating the complex number array that helps run Santa's sleigh. The boss discovered that the last development team stored the arrays using Cartesian coordinates rather than polar coordinates, which everyone knows are the boss' favorite.
+
+For this task you will need to implement the function `cartesianToPolar(_:)` that takes in a tuple, `(x: Double, y: Double)`, and return a new tuple, `(r: Double, theta: Double)`, where:
+
+- r = √(x<sup>2</sup> + y<sup>2</sup>)
+- theta = _atan2_(y, x)
```

Comment:
@iHiD What do you think about this math-based task? It is fairly light on math, but I'd appreciate a second look :)

Reply:
- Could we use more meaningful variable names that aren't `r` and `theta`?
- Have we taught how to do sqroot somewhere?
- I don't know what `atan` is so that needs explaining if its not.
- cc @SaschaMann and @TheLostLambda for thoughts and mathsjax or whatever its called that makes formulas look pretty.

Reply:
```suggestion
- $r = \sqrt{x^2+y^2}$
- $\vartheta = \operatorname{atan2}(y, x)$
```

I'm not sure about the delimiters that would be used (since I don't know how the markdown is translated to HTML), but I used `$` and `$$` as placeholders until we can test it. This is rendered as follows:
![grafik](https://user-images.githubusercontent.com/20866761/91670058-2a534a00-eb1a-11ea-8073-95d7ad8b0aa2.png)

---

Side note: I'd suggest adding the definition of `atan2` somewhere to the instructions, I had to look it up. You can copy the LaTeX source from the Wikipedia source here if you want to save time: https://en.wikipedia.org/wiki/Polar_coordinate_system#Converting_between_polar_and_Cartesian_coordinates

> Could we use more meaningful variable names that aren't r and theta?

In my unsolicited opinion: in this context, any other variable names would be less meaningful.

Reply:
It was not obvious to me that `r` meant `radius` and I've never come across `theta` in my life before. 

Based on the fact I'm pretty well educated (with a Comp Sci degree) I would contest that most people that haven't studied maths to some more significant (and/or recent) extent would remember or have come in contact with this 🙂

Reply:
Sorry, let me clarify. Those who have seen polar coordinates or similar geometric constructs before will expect those names. For those that haven't, either the name itself doesn't matter, they can accept it as it is and just plug it into the formula, or, if they want to fully understand the idea behind them, they will benefit from the canonical names because all resources will use them. If we use something else they'd have to translate between the exercise intro and the 3b1b video they're watching. For the same reason we call a tuple a tuple and not a collectionOfElementsWithDifferentTypes, even if the name tuple is meaningless to some people.

Reply:
One possible solution would be to mention somewhere that `r` is the radius.

Reply:
Or include a picture: https://en.wikipedia.org/wiki/Polar_coordinate_system#/media/File:Polar_to_cartesian.svg

Reply:
@SaschaMann :+1: